kythe/go/extractors/cmd/bazel/extract_kzip/extract_kzip.go

```diff
@@ -25,7 +25,6 @@ import (
 	"time"
 
 	"kythe.io/kythe/go/extractors/bazel"
-	"kythe.io/kythe/go/extractors/bazel/extutil"
```

Comment:
Is there a corresponding BUILD dependency to prune?

Reply:
Ah you're right. Removed unused dependency